_episodes/12-large-files.md

```diff
@@ -3,14 +3,164 @@ title: "Large Files"
 teaching: 0
 exercises: 0
 questions:
-- "How do I handle large files in git?"
+- "Why are (large) binary files a problem in Git?"
+- "What is Git LFS?"
+- "What are the problems with Git LFS?"
 objectives:
-- "First learning objective. (FIXME)"
+- "Understanding that Git is not intended for (large) binary files"
+- "Learning about the `git lfs` commands"
+- "Understanding the disadvantages of `git lfs`"
 keypoints:
-- "First key point. Brief Answer to questions. (FIXME)"
+- "(Large) binary files can grow the repository size immensely and make it unusable"
+- "`git lfs` is an extension that stores large files outside the Git data model"
+- "Use of Git LFS is discouraged in many scenarios."
 ---
 
+TODO: Installation steps for this episode
```

Comment:
Can you please add a link to this set of installation instructions for completeness: https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage

Reply:
Added to the respective issue #26